code/__HELPERS/globalaccess.dm

```diff
@@ -1751,8 +1751,6 @@
 			return global.GPS_list;
 		if("SPS_list")
 			return global.SPS_list;
-		if("secure_GPS_count")
```

Comment:
before this PR it was used by the SPS to increase the number after the tag (e.g. SEC13)
with this PR I extended that behaviour to the regular GPSs, so now normal ones have their own counter and sec keeps theirs as well.
The way the number is determined is through `gen_id()`, which merely returns the current GPS/SPS_list's `len`.

basically, yes, unused